HomeTask2/Program.cs

```diff
@@ -4,12 +4,67 @@
 using System.Text;
 using System.Threading.Tasks;
 
+
+
 namespace HomeTask2
 {
+
+    
+
     class Program
     {
-        static void Main(string[] args)
+
+                static void Main(string[] args)
         {
+            // User input nominal value
+            Console.WriteLine("Please insert the nominal");
+            var fnominal = Console.ReadLine();
+            var nominal = double.Parse(fnominal);
+            Console.WriteLine($"Nominal is:  {nominal}");
+
+            // User input trade price
+            Console.WriteLine("Please insert the trade price");
+            var ftradePrice = Console.ReadLine();
```

Comment:
I think in this case inputString or inputPrice is more descriptive name then ftradePrice